lib/workers/repository/init/vulnerability.spec.ts

```diff
@@ -367,40 +367,5 @@ describe('workers/repository/init/vulnerability', () => {
       expect(res.packageRules).toMatchSnapshot();
       expect(res.packageRules).toHaveLength(1);
     });
-
-    it('returns pip alerts with normalized name', async () => {
```

Comment:
no longer needed to validate?

I think it can still fail osv?

Reply:
Yes, because I expect that `[github.getVulnerabilityAlerts](https://github.com/renovatebot/renovate/pull/27733/commits/a1fbc42083aa469af60b78ddca8ff66f64355972)` returns normalized names. So the mock here doesn't make sense. Maybe an error could be thrown in `workers/repository/init/vulnerability`?